ui/src/ui/pages/Signup/Signup.tsx

```diff
@@ -0,0 +1,176 @@
+    if (emailError || passwordError) {
+      return;
+    }
+    const res = await signUpUser(formData);
+    const response = (
+      'data' in res ? res.data : 'data' in res.error ? res.error.data : null
+    );
+
+    if (response.success) {
+      toast.success(`${response.message}`, {
+        duration: 3000,
+        position: 'bottom-center',
+        ariaProps: {
+          'role': 'status',
+          'aria-live': 'polite',
+        },
+      });
+      navigate('/dashboard');
+    } else {
+      toast.error(`${response.message}`, {
+        duration: 3000,
+        position: 'bottom-center',
+        ariaProps: {
+          'role': 'status',
+          'aria-live': 'polite',
+        },
+      });
+    }
+  };
+
+  return (
+    <section className="container">
+      <div className="form-container">
+        <div className='flex-column-center h-full'>
+          <img src="/assets/logo.png" alt="" width={150}/>
+          <h2 className='form-title'>Create an Account</h2>
+        </div>
+        <form className='flex-column-between w-full h-full'>
+          <div className='w-full'>
+            <InputField
+              title={'Email Address'}
+              id={'email'}
+              type={'text'}
+              onChange={(e) => setFormData({ ...formData, [e.target.id]: e.target.value })}
+              value={email}
+              placeHolder={'Your Email Address'}
+              Icon={AiOutlineUser}
+              disabled={false}
+            />
+            <div className={`error-message-container ${emailErrorMessage ? 'visible' : 'hidden'}`}>
+              <BiError className="error-message-icon" />
+              <p className="error-message">{emailErrorMessage}</p>
+            </div>
+          </div>
+          <InputField
+            title={'User Name'}
+            id={'displayName'}
+            type={'text'}
+            onChange={(e) => setFormData({ ...formData, [e.target.id]: e.target.value })}
+            value={displayName}
+            placeHolder={'Your User Name'}
+            Icon={CgNametag}
+            disabled={false}
+          />
+          <div className='w-full'>
+            <PasswordField
+              title={'Password'}
+              id={'password'}
+              onChange={(e) => setFormData({ ...formData, [e.target.id]: e.target.value })}
+              value={password}
+              placeHolder={'Your Password'}
+              Icon={AiOutlineLock}
+              disabled={false}
+            />
+            <div className={`error-message-container ${passwordErrorMessage ? 'visible' : 'hidden'}`}>
+              <BiError className="error-message-icon" />
+              <p className="error-message">{passwordErrorMessage}</p>
+            </div>
+          </div>
+          <div className='flex-row-between'>
+            <SimpleLink link={'/signin'} title={'Already a member? Signin!'} color='blue' />
+            <SimpleLink link={'/forgotpassword'} title={'Forgot Password?'} color='red'/>
```

Comment:
`'/forgotpassword'` move this to constant

Reply:
Done